CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# Code of Conduct for vCon Open Source Project
+
+## 1. Introduction
+
+As contributors and maintainers of the vCon open source project, we are committed to fostering an inclusive, welcoming, and respectful community. This Code of Conduct outlines the expectations for all members of our community, as well as the consequences for unacceptable behavior.
+
+We invite all participants in our community to help us create a safe and positive environment for everyone.
+
+## 2. Our Standards
+
+Examples of behavior that contributes to a positive environment include:
+
+- Using inclusive language and avoiding gendered terms.
+- Being respectful of differing viewpoints and experiences.
+- Gracefully accepting constructive criticism.
+- Focusing on what is best for the community.
+- Showing empathy towards other community members.
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or advances.
+- Trolling, insulting/derogatory comments, and personal or political attacks.
+- Public or private harassment.
+- Publishing others' private information, such as a physical or email address, without explicit permission.
+- Other conduct that could reasonably be considered inappropriate in a professional setting.
+
+## 3. Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct, or to temporarily or permanently ban any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+## 4. Scope
+
+This Code of Conduct applies to all participants in the vCon open source project, including contributors, maintainers, and users. It also applies to all project-related spaces, such as issue trackers, pull requests, code reviews, and any other online or offline communication channels.
+
+## 5. Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [vcon-conduct@example.com]. All complaints will be reviewed and investigated, and the project team will determine the appropriate course of action at their discretion. The project team is obligated to maintain confidentiality concerning the reporter of an incident.
```

Comment:
Need to change example.com to vcon-dev.org??

Do we have vcon-dev setup with email.  I don't think we need anything other than aliases to redirect/forward email.  I would avoid setting up email boxes, etc.